css-ecommerce-page/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Ecommerce Challenge</title>
+  <link href="https://fonts.googleapis.com/css?family=DM+Sans&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="header">
+    <header>
+      <p>New Arrivals</p>
+    </header>
+  </div>
+  <div class="container">
+    <div class="row">
+      <div class="col-2">
+        <img src="images/dress1.jpeg" alt="dress1">
+      </div>
+      <div class="col-2">
+        <img src="images/ysl-bag.jpeg" alt="dress1">
+      </div>
+      <div class="col-2">
+        <img src="images/key.jpeg" alt="dress1">
+      </div>
+      <div class="col-2">
+        <img src="images/lv-bag.jpg" alt="dress1">
+      </div>
+      <div class="col-2">
+        <img src="images/dress2.jpeg" alt="dress1">
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-2">
+        <p>Free people <span> |Size 6</span></p>
+      </div>
+      <div class="col-2">
+        <p>Yves Saint Laurent <span> |One-Size</span></p>
+      </div>
+      <div class="col-2">
+        <p>Tiffany <span> |One-Size</span></p>
+      </div>
+      <div class="col-2">
+        <p>Louis Vuitton <span>|One-Size</span></p>
+      </div>
+      <div class="col-2">
+        <p>Reformation <span> |Size 4</span></p>
+      </div>
+    </div>
```

Comment:
There should be more space between the separator and the size. Also, the separator should be taller relative to the text:

---

![image](https://user-images.githubusercontent.com/16143771/152212722-668a4b96-8174-4062-a3f2-f0f69d94c555.png)

---

![image](https://user-images.githubusercontent.com/16143771/152212762-e72008ba-b7e0-4047-a0e1-0819cb2031fc.png)